cla_public/templates/base.html

```diff
@@ -84,6 +118,36 @@
 
 {% block javascripts %}
   <script>
+
+    var currentLocation = window.location.href;
+    if (currentLocation.indexOf("n43n3") > 0 || currentLocation.indexOf("n43n14/n97") > 0 || currentLocation.indexOf("n88") > 0 || currentLocation.indexOf("n149") > 0 || currentLocation.indexOf("n86") > 0) {
+      $(".laa-flee-button-container").addClass("laa-fee-button-enabled");
+    }
+
+    function flee() {
+      $("body").hide();
+      {#
+        The below are all other possibilities but not necessarily the best approach. 
+
+        var histLength = history.length;
+        window.history.go(-histLength);
+        history.pushState({page: 1}, "Cookies", "/cookies")
+        history.pushState({page: 2}, "Bank Holidays in the UK", "/bank-holidays")
+      #}
+      window.open("http://bbc.co.uk/weather", "_newtab");
+      window.location.replace("http://www.google.co.uk");
```

Comment:
Would the following be suitable here:
```
window.open("http://bbc.co.uk/weather", "_newtab");
window.close()
```
Opens bbc weather in a new tab and closes the current tab. Stops the browser back button being used to find previous site

Reply:
This doesn't work unless the page has been opened with JS.  It is one of the methods we can perhaps look at long term but it will mean a few changes elsewhere in the system.  

Although `window.close()` works when typed directly into the console, try adding it to a button `<button onclick="window.close();">Close Window</button>` and it won't work.

Reply:
Yep, I had the idea but only tested it on the console. Ignore me